tsl/test/expected/cagg_api.out

```diff
@@ -0,0 +1,159 @@
+  JOIN _timescaledb_catalog.continuous_agg
+    ON hypertable.id = continuous_agg.mat_hypertable_id;
+CREATE TABLE hyper_ts (time timestamp NOT NULL, value float);
+CREATE TABLE hyper_tstz (time timestamptz NOT NULL, value float);
+CREATE TABLE hyper_multi (time timestamptz NOT NULL, device int, value float);
+CREATE TABLE hyper_no_cagg (time timestamptz NOT NULL, device int, value float);
+CREATE TABLE normal_ts(time timestamp NOT NULL, value float);
+SELECT * FROM create_hypertable('hyper_ts', 'time');
+WARNING:  column type "timestamp without time zone" used for "time" does not follow best practices
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             1 | public      | hyper_ts   | t
+(1 row)
+
+SELECT * FROM create_hypertable('hyper_tstz', 'time');
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             2 | public      | hyper_tstz | t
+(1 row)
+
+SELECT * FROM create_hypertable('hyper_no_cagg', 'time');
+ hypertable_id | schema_name |  table_name   | created 
+---------------+-------------+---------------+---------
+             3 | public      | hyper_no_cagg | t
+(1 row)
+
+SELECT * FROM create_hypertable('hyper_multi', 'time', 'device', 4);
+ hypertable_id | schema_name | table_name  | created 
+---------------+-------------+-------------+---------
+             4 | public      | hyper_multi | t
+(1 row)
+
+INSERT INTO hyper_ts
+  SELECT time, ceil(random() * 100)::int
+    FROM generate_series('2025-01-01'::timestamp,
+                         '2025-01-06', '1m') time;
+INSERT INTO hyper_tstz
+  SELECT time, ceil(random() * 100)::int
+    FROM generate_series('2025-01-01'::timestamptz,
+                         '2025-01-06', '1m') time;
+CREATE MATERIALIZED VIEW ts_temperature_1h
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('1 hour', time), avg(value)
+  FROM hyper_ts
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "ts_temperature_1h"
+CREATE MATERIALIZED VIEW ts_temperature_15m
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('15 minutes', time), avg(value)
+  FROM hyper_ts
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "ts_temperature_15m"
+CREATE MATERIALIZED VIEW tstz_temperature_1h
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('1 hour', time), avg(value)
+  FROM hyper_tstz
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "tstz_temperature_1h"
+CREATE MATERIALIZED VIEW tstz_temperature_15m
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('15 minutes', time), avg(value)
+  FROM hyper_tstz
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "tstz_temperature_15m"
+CREATE MATERIALIZED VIEW multi_temperature_15m
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('15 minutes', time), avg(value)
+  FROM hyper_multi
+GROUP BY 1;
+NOTICE:  continuous aggregate "multi_temperature_15m" is already up-to-date
```

Comment:
Is it expected that this cagg is already up-to-date at creation?

Reply:
The error message is a bit strange, I agree, but since it is empty this is expected.